kuma/javascript/ssr-server.js

```diff
@@ -48,6 +33,28 @@ const MAX_BODY_SIZE = 1024 * 1024 * 5; // 5 megabyte max JSON payload
 // We're using the Express server framework
 const app = express();
 
+// Configure and initialize Sentry if a DSN has been provided.
+if (process.env.SENTRY_DSN) {
+    console.log('Configuring Sentry for the SSR server.');
+    let options = {
+        dsn: process.env.SENTRY_DSN
+    };
+    if (process.env.REVISION_HASH) {
+        options.release = process.env.REVISION_HASH;
+    }
+    if (process.env.SENTRY_ENVIRONMENT) {
+        options.environment = process.env.SENTRY_ENVIRONMENT;
+    }
+    Sentry.init(options);
+
+    // The request handler must be the first middleware on the app
+    app.use(Sentry.Handlers.requestHandler());
+    // The error handler must be before any other error middleware
+    app.use(Sentry.Handlers.errorHandler());
```

Comment:
These 4 lines right here (above) is the only new thing. The rest is that I just moved this stuff to be *AFTER* the `const app = express();` (line 34). 
If you don't do this you don't get information in Sentry about what what posted to SSR.

Reply:
As you say in the comment, this must be before any other error middleware, but it also must be **after** any Express controllers you'd like it to monitor/handle. If you move this below the definition of the `app.post` controller below, you'd no longer need to explicitly capture exceptions and send them to Sentry (I tested it locally, and it works). The downside, though, is that it would monitor/handle all of the other controllers as well, unless you define the `app.post` controller first, do this `app.use(Sentry.Handlers.errorHandler());` next, and finally add all of the remaining Express controllers (e.g., the `healthz` and `readiness` endpoints) that we don't really care to have Sentry handle.

Reply:
Resolved by discussion.